tests/testsuite.py

```diff
@@ -197,6 +196,12 @@ def test_one_import(self):
         m = b.struct_mystruct()
         b.bar(ctypes.byref(m))
 
+    # NOTE(geisserml) test_two_import passed before (while test_two_import_with_embedded_preamble
+    # already failed). Not sure why. Anyway, if we submit a struct of module 'b' to a function of
+    # module 'a', it's quite logical that ctypes raises a type error. Why were the tests written
+    # like this in the first place?
+
+    @unittest.expectedFailure
```

Comment:
I think that this didn't fail in the fist place might have been a side effect of the `POINTER` monkeypatch (see above)?